### PR TITLE
SEO-176838-vue-ej2-page-with-redirect

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes to the redirect links,
Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: ej2-vue/card/getting-started.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ yarn add @syncfusion/ej2-vue-layouts
 
 ## Import Syncfusion CSS styles
 
-You can import themes for the Syncfusion Vue component in various ways, such as using CSS or SASS styles from npm packages, CDN, [CRG](https://ej2.syncfusion.com/javascript/documentation/common/custom-resource-generator/) and [Theme Studio](https://ej2.syncfusion.com/vue/documentation/appearance/theme-studio/). Refer to [themes topic](https://ej2.syncfusion.com/vue/documentation/appearance/theme/) to know more about built-in themes and different ways to refer to themes in a Vue project.
+You can import themes for the Syncfusion Vue component in various ways, such as using CSS or SASS styles from npm packages, CDN, [CRG](https://ej2.syncfusion.com/javascript/documentation/common/custom-resource-generator/) and [Theme Studio](https://ej2.syncfusion.com/vue/documentation/appearance/theme-studio). Refer to [themes topic](https://ej2.syncfusion.com/vue/documentation/appearance/theme) to know more about built-in themes and different ways to refer to themes in a Vue project.
```
Comment (Yvone-Atieno) [inline ej2-vue/card/getting-started.md:70]: Check CRG link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-vue/card/getting-started.md (hunk 1 above)
Comment (Edithabich) [reply ej2-vue/card/getting-started.md:70]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**